packages/iris-grid/src/sidebar/VisibilityOrderingBuilder.tsx

```diff
@@ -901,133 +1011,125 @@ class VisibilityOrderingBuilder extends Component<
-              kind="ghost"
-              className="btn-link-icon"
-              onClick={() => {
-                this.handleMoveColumns(
-                  VisibilityOrderingBuilder.MOVE_OPTIONS.TOP
-                );
-              }}
-              disabled={noSelection}
-              tooltip="Move selection to top"
-            >
-              <FontAwesomeIcon icon={dhArrowToTop} />
-            </Button>
-            <Button
-              kind="ghost"
-              className="btn-link-icon"
-              onClick={() => {
-                this.handleMoveColumns(
-                  VisibilityOrderingBuilder.MOVE_OPTIONS.BOTTOM
-                );
-              }}
-              disabled={noSelection}
-              tooltip="Move selection to bottom"
-            >
-              <FontAwesomeIcon icon={dhArrowToBottom} />
-            </Button>
-          </div>
-        </div>
-        <div className="top-menu">
           <SearchInput
-            className="w-100"
+            className="visibility-search"
             value={searchFilter}
             matchCount={searchFilter ? selectedColumns.length : undefined}
             onChange={this.handleSearchInputChange}
           />
         </div>
-        <DragDropContext
-          onDragStart={this.handleOrderDraggingStart}
-          onDragEnd={this.handleOrderDraggingEnd}
-        >
-          <Droppable droppableId="droppable-visibility-order-list">
-            {(provided, snapshot) => (
-              // eslint-disable-next-line jsx-a11y/interactive-supports-focus
-              <div
-                role="menu"
-                className={classNames('visibility-ordering-list', {
-                  isDragging: snapshot.draggingFromThisWith,
-                })}
-                ref={provided.innerRef}
-                // eslint-disable-next-line react/jsx-props-no-spreading
-                {...provided.droppableProps}
-              >
-                <div
-                  className="column-list"
-                  ref={list => {
-                    this.list = list;
-                  }}
-                >
-                  {columnItems}
-                  {provided.placeholder}
-                </div>
-              </div>
-            )}
-          </Droppable>
-        </DragDropContext>
+        <div className="top-menu">
+          <Button
+            kind="ghost"
+            icon={vsRefresh}
+            tooltip="Reset to default"
+            onClick={() => {
+              this.resetVisibilityOrdering();
+            }}
+          >
+            Reset
+          </Button>
+          <span className="vertical-divider" />
+          <Button
+            kind="ghost"
+            tooltip={
+              selectedColumns.length === 0
+                ? 'Select items to create group'
+                : 'Create group'
+            }
+            disabled={selectedColumns.length === 0}
+            onClick={() => this.handleGroupCreate(selectedColumns)}
```

Comment:
Not sure why you're doing this instead of just passing `onClick={this.handleGroupCreate}` and get `selectedColumns` from state in there.